src/components/Header/HeaderDescription.tsx

```diff
@@ -36,10 +36,10 @@ class HeaderDescription extends UIComponent<ReactProps<HeaderDescriptionProps>,
     as: 'p',
   }
 
-  renderComponent({ ElementType, classes, unhandledProps }) {
+  renderComponent({ ElementType, classes, unhandledProps, accessibility }) {
     const { children, content } = this.props
     return (
-      <ElementType {...unhandledProps} className={classes.root}>
+      <ElementType {...unhandledProps} {...accessibility.attributes.root} className={classes.root}>
```

Comment:
we should put `unhandledProps` as the last ones there, as otherwise there won't be any possibility for overrides opened to the client